src/res.rs

```diff
@@ -249,12 +251,22 @@ impl Resources {
     pub fn fetch<T>(&self, id: usize) -> Fetch<T>
         where T: Resource
     {
-        let c = self.fetch_internal(TypeId::of::<T>(), id);
+        self.try_fetch(id).expect(RESOURCE_NOT_FOUND)
```

Comment:
I wanted to avoid duplication of the code that builds the `Fetch`. Do you want me to leave it the way it was?